plugins/node/instrumentation-lru-memoizer/README.md

```diff
@@ -17,7 +17,7 @@ npm install --save @opentelemetry/instrumentation-lru-memoizer
 
 ## Supported Versions
 
-- `>=1.3 <3`
+- [`lru-memorizer`](https://github.com/jfromaniello/lru-memoizer) versions `>=1.3 <3`
```

Comment:
```suggestion
- [`lru-memorizer`](https://www.npmjs.com/package/lru-memoizer) versions `>=1.3 <3`
```

Reply:
Thanks, I was copying this text from the text in the beginning of the README. makes sense to align it to `npm` at least for the new additions